index.js

```diff
@@ -1,95 +1,139 @@
 // ***** Question 1 *****
 
 // *** Uncomment the lines below to test
-// console.log("%cQuestion 1", "color: red") 
 
-// console.log(counter) 
-// // => 2
-// console.log("%c----------", "color: red") 
+counter = 1
+counter += 1
```

Comment:
Be careful on line 5 here, if you declare a variable without let or const it will become a global variable (even if you declare it inside a function). Always use `let` or `const` to create new variables.